arangod/RocksDBEngine/RocksDBTransactionCollection.cpp

```diff
@@ -45,7 +46,10 @@ RocksDBTransactionCollection::RocksDBTransactionCollection(TransactionState* trx
       _numInserts(0),
       _numUpdates(0),
       _numRemoves(0),
-      _usageLocked(false) {}
+      _usageLocked(false),
+      _exclusiveWrites(trx->vocbase().server().getFeature<arangodb::RocksDBOptionFeature>()._exclusiveWrites)
+      {}
```

Comment:
why is it necessary to have this bool in every single transaction ?

Reply:
How would you do it?